### PR TITLE
Updated to monaco-vscode-api@1.83.10. Align with changes

### PR DESCRIPTION
**Pull request by kaisalmen**
Looking good apart from this:
image

It is called from inside `monaco.js`: 
image

### CODE REVIEW

**Comment by CGNonofr** [comment]
Are you sure there is no duplicated package?

**Comment by kaisalmen** [comment]
Observation: Updating the overrides usually requires to nuke both `package-lock.json` and `node_modules` otherwise you get mismatching `monaco-editor` versions (e.g. `0.44.0` suddenly bleeding in again)

> Are you sure there is no duplicated package?

At least not `monaco-editor`. I will check if there is something else wrong.

**Comment by CGNonofr** [comment]
I would be surprised there is that error here and not in monaco-vscode-api demo.

**Comment by kaisalmen** [comment]
It happens when I use `getKeybindingsServiceOverride`. If I comment that one the error is gone. Did I miss a change in that regard?

**Comment by CGNonofr** [comment]
It doesn't ring a bell to me

**Comment by kaisalmen** [comment]
> It doesn't ring a bell to me

I haven't found a smoking gun either last night, but I also didn't search deeper. I wanted to verify in the demo and discovered https://github.com/CodinGame/monaco-vscode-api/issues/259

**Comment by kaisalmen** [comment]
@CGNonofr `monaco.js` is imported from both packages:
image
The later one is called first and the second one then results in the error.

**Comment by kaisalmen** [comment]
@CGNonofr Works fine now! Thank you. Will push an update soon.

**Review by CGNonofr — APPROVED** [review]

File: packages/client/src/monaco-vscode-api-services.ts [hunk 1]
```diff
@@ -98,7 +98,7 @@ export const importAllServices = async (config?: InitializeServiceConfig) => {
 
     const haveThemeService = serviceNames.includes('theme') || Object.keys(userServices).includes('themeService');
     const haveTextmateService = serviceNames.includes('textmate') || Object.keys(userServices).includes('textMateTokenizationFeature');
-    const haveMarkersService = serviceNames.includes('markers');
+    const haveMarkersService = serviceNames.includes('markers') || Object.keys(userServices).includes('markersService');
```
[the remark this answers is not in the document]
Comment (kaisalmen) [reply packages/client/src/monaco-vscode-api-services.ts:101]: > Isn't serviceNames always empty here though?

Thanks, I didn't remove that when removing the conditional imports a couple of releases ago.

File: packages/client/src/monaco-vscode-api-services.ts [hunk 2]
```diff
@@ -112,7 +112,8 @@ export const importAllServices = async (config?: InitializeServiceConfig) => {
     }
 
     await initialize(userServices);
-    if (lc.logLevel) {
-        StandaloneServices.get(ILogService).setLevel(lc.logLevel);
+    const logLevel = lc.workspaceConfig?.developmentOptions?.logLevel;
+    if (logLevel) {
+        StandaloneServices.get(ILogService).setLevel(logLevel);
```
[the remark this answers is not in the document]
Comment (kaisalmen) [reply packages/client/src/monaco-vscode-api-services.ts:117]: But by defaults already logs to the console. There are one or two examples where debug level is configured and you can see it. Let's improve this with another PR (if required), ok?

File: packages/client/src/monaco-vscode-api-services.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (kaisalmen) [reply packages/client/src/monaco-vscode-api-services.ts]: Updated in next commit ⬇️

**Pull request merged**